pkg/gui/gui.go

```diff
@@ -382,7 +382,7 @@ func initialWindowViewNameMap(contextTree *context.ContextTree) *utils.ThreadSaf
 
 func initialScreenMode(startArgs appTypes.StartArgs, config config.AppConfigurer) types.WindowMaximisation {
 	if startArgs.FilterPath != "" || startArgs.GitArg != appTypes.GitArgNone {
-		return types.SCREEN_HALF
+		return types.SCREEN_FULL
```

Comment:
This is a separate thing: I think if you do `lazygit log` you should just see the fullscreened commits view rather than half-sized